plugins/agents/ping/Ping.go

```diff
@@ -0,0 +1,113 @@
+package ping
+
+import (
+	"strings"
+	"time"
+
+	"github.com/abrander/agento/plugins"
+	"github.com/abrander/agento/timeseries"
+	gping "github.com/gansoi/gansoi/plugins/agents/ping"
```

Comment:
Why the `gping`-prefix? Why not just just the package name from Gansoi? (`ping`)

Reply:
@abrander to avoid misunderstanding it - the module is called `Ping` and uses that name, so to make it clear that the `gping` is something from the outside. If you don't like it, I'll change it...

Reply:
@anderskvist I can see the reasoning, I got confused and looked for *another* import using the ping keyword. Up to you :-)

Reply:
Changed `gping` to `gansoiPing` to make it easier to understand